changelog.json

```
@@ -1,5 +1,17 @@
 {
   "releases": {
+    "2.7.3": [
```

Shouldn't this be 2.8.0?
```suggestion
    "2.8.0": [
```